src/Identity/UI/src/IdentityDefaultUIConfigureOptions.cs

```diff
@@ -24,7 +24,7 @@ internal class IdentityDefaultUIConfigureOptions<TUser> :
 
     public IWebHostEnvironment Environment { get; }
 
-    public void PostConfigure(string name, RazorPagesOptions options)
+    public void PostConfigure(string? name, RazorPagesOptions options)
     {
         name = name ?? throw new ArgumentNullException(nameof(name));
```

Comment:
💭 Well this is concerning. Maybe add a comment describing why the API says null is allowed but this method doesn't allow it?

📝 Normally the exception for an input restriction added by a derived type would be `NotSupportedException` (e.g. how read only collections handle mutating methods defined by `IList<T>`.

Reply:
I think we could probably remove this check entirely. The parameter doesn't get used at all, so I'm not sure why throwing when it is null is meaningful.

@HaoK, thoughts? /cc @BrennanConroy since you did something similar recently.

Reply:
name is allowed to be null, we should remove it, this is a subtle bug

Reply:
```suggestion
```

Reply:
Yeah, I left the ones that didn't use `name` or used it in a safe way alone, and the ones that used `name` but expected a value I added the throw.